.github/workflows/test.yml

```diff
@@ -42,79 +42,132 @@ jobs:
             pg: '10'
             pgis: '2.4'
             perl: '5.24'
+            release: bionic
           - os: ubuntu-18.04
             pg: '10'
             pgis: '2.5'
             perl: '5.24'
+            release: bionic
           - os: ubuntu-18.04
             pg: '10'
             pgis: '3'
             perl: '5.24'
+            release: bionic
           - os: ubuntu-18.04
             pg: '12'
             pgis: '2.5'
             perl: '5.24'
+            release: bionic
           - os: ubuntu-18.04
             pg: '12'
             pgis: '3'
             perl: '5.24'
+            release: bionic
           - os: ubuntu-18.04
             pg: '10'
             pgis: '2.4'
             perl: '5.26'
+            release: bionic
           - os: ubuntu-18.04
             pg: '10'
             pgis: '2.5'
             perl: '5.26'
+            release: bionic
           - os: ubuntu-18.04
             pg: '10'
             pgis: '3'
             perl: '5.26'
+            release: bionic
           - os: ubuntu-18.04
             pg: '12'
             pgis: '2.5'
             perl: '5.26'
+            release: bionic
           - os: ubuntu-18.04
             pg: '12'
             pgis: '3'
             perl: '5.26'
+            release: bionic
 
     env:
       # Set PATH as postgresql-server-dev-all pretends version is 11
       PATH: /usr/lib/postgresql/${{ matrix.pg }}/bin:/bin:/usr/bin:/usr/local/bin
-      APT: 'apt-get -o Dpkg::Progress=0 -o Dpkg::Use-Pty=0'
-      PCLOUD: https://packagecloud.io/linz
     steps:
       - name: Check out repository
         uses: actions/checkout@v2.4.0
         with:
           fetch-depth: 0
 
-      - name: Install dependencies
+      - name: Remove conflicting packages
+        run: |
+          sudo apt-get --assume-yes -qq purge \
+          libpq* \
+          *php* \
+          postgresql*
+
+      - name: Remove PostgreSQL install files
+        run: sudo rm --force --recursive /var/lib/postgresql
+
+      - name: Update package database
+        run: sudo apt-get update
+
+      - name: Install setup dependencies
+        run: sudo apt-get --assume-yes install curl ca-certificates gnupg
+
+      - name: Get release codename
```

Comment:
You shouldn't need to do this in this job - there's already `${{ matrix.release }}`.